File: mvp-widget/src/main/java/ru/surfstudio/android/mvp/widget/event/delegate/WidgetScreenEventDelegateManager.java
package ru.surfstudio.android.mvp.widget.event.delegate;


import java.util.List;

import androidx.annotation.Nullable;
import ru.surfstudio.android.core.ui.ScreenType;
import ru.surfstudio.android.core.ui.event.BaseScreenEventDelegateManager;
import ru.surfstudio.android.core.ui.event.ScreenEventDelegateManager;
import ru.surfstudio.android.core.ui.event.base.resolver.ScreenEventResolver;

public class WidgetScreenEventDelegateManager extends BaseScreenEventDelegateManager {

    public WidgetScreenEventDelegateManager(List<ScreenEventResolver> eventResolvers,
                                            @Nullable ScreenEventDelegateManager parentDelegateManger) {
        super(eventResolvers, parentDelegateManger, ScreenType.WIDGET);
    }
}

File: mvp-widget/src/main/java/ru/surfstudio/android/mvp/widget/event/delegate/WidgetScreenEventDelegateManager.kt
package ru.surfstudio.android.mvp.widget.event.delegate

import ru.surfstudio.android.core.ui.ScreenType
import ru.surfstudio.android.core.ui.event.ScreenEventDelegateManager
import ru.surfstudio.android.core.ui.event.base.ScreenEvent
import ru.surfstudio.android.core.ui.event.base.ScreenEventDelegate

class WidgetScreenEventDelegateManager : ScreenEventDelegateManager{

    override fun registerDelegate(delegate: ScreenEventDelegate?) {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun registerDelegate(delegate: ScreenEventDelegate?, emitterType: ScreenType?) {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun unregisterDelegate(delegate: ScreenEventDelegate?): Boolean {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun <E : ScreenEvent?, D : ScreenEventDelegate?, R : Any?> sendEvent(event: E): R {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun destroy() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

}